<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Glicko-2 Ranker</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="mainLay" stretch="1,3,1">
      <item>
       <widget class="QListWidget" name="teamList">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>1</width>
          <height>3</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>1</width>
          <height>3</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="infoLay" stretch="2,3">
        <item>
         <layout class="QHBoxLayout" name="sysInfoLay">
          <item>
           <layout class="QVBoxLayout" name="sysValLay">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="numTeamsLay" stretch="0,0">
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="numTeamsLabel">
                <property name="minimumSize">
                 <size>
                  <width>140</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>11</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Number of Teams:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="numTeamsView">
                <property name="minimumSize">
                 <size>
                  <width>50</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>11</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>32</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="numMatchesLay">
              <property name="spacing">
               <number>5</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="numMatchesLabel">
                <property name="minimumSize">
                 <size>
                  <width>140</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>11</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Number of Matches Complete:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="numMatchesView">
                <property name="minimumSize">
                 <size>
                  <width>50</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>11</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>32</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="sysConLay">
              <item>
               <widget class="QLabel" name="sysConLabel">
                <property name="minimumSize">
                 <size>
                  <width>140</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>11</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>System Constant:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="sysConView">
                <property name="text">
                 <string>0.8</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="ratingRangeLay">
              <item>
               <widget class="QLabel" name="ratingRangeLabel">
                <property name="minimumSize">
                 <size>
                  <width>140</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>11</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Range of Ratings:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="ratingRangeView">
                <property name="minimumSize">
                 <size>
                  <width>50</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>11</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>500-3500</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QChartView" name="rankDistChart"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="teamInfoLay">
          <item>
           <widget class="QLabel" name="teamNameLabel">
            <property name="font">
             <font>
              <pointsize>20</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Team Name Here</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="teamGraphInfoLay">
            <item>
             <layout class="QVBoxLayout" name="teamTextInfoLay">
              <item>
               <layout class="QHBoxLayout" name="ratingLay">
                <item>
                 <widget class="QLabel" name="ratingLabel">
                  <property name="minimumSize">
                   <size>
                    <width>140</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>11</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Rating:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="ratingView">
                  <property name="minimumSize">
                   <size>
                    <width>140</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>11</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>2753</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="ratingDevLay">
                <item>
                 <widget class="QLabel" name="ratingDevLabel">
                  <property name="minimumSize">
                   <size>
                    <width>140</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>11</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Rating Deviation:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="ratingDevView">
                  <property name="minimumSize">
                   <size>
                    <width>140</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>11</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>351</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="volLay">
                <item>
                 <widget class="QLabel" name="volLabel">
                  <property name="minimumSize">
                   <size>
                    <width>140</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>11</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Volatility:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="volView">
                  <property name="minimumSize">
                   <size>
                    <width>140</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>11</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>0.08</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="matWonLay">
                <item>
                 <widget class="QLabel" name="matWonLabel">
                  <property name="minimumSize">
                   <size>
                    <width>140</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>11</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Matches Won:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="matWonView">
                  <property name="minimumSize">
                   <size>
                    <width>140</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>11</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>12</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="matLostLay">
                <item>
                 <widget class="QLabel" name="matLostLabel">
                  <property name="minimumSize">
                   <size>
                    <width>140</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>11</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Matches Lost:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="matLostView">
                  <property name="minimumSize">
                   <size>
                    <width>140</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>11</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>12</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="matDrawnLay">
                <item>
                 <widget class="QLabel" name="matDrawnLabel">
                  <property name="minimumSize">
                   <size>
                    <width>140</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>11</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Matches Drawn:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="matDrawnView">
                  <property name="minimumSize">
                   <size>
                    <width>140</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>11</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>1</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QChartView" name="rateHistoryView"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="matchList"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>790</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionSystem_Constant"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuTeams">
    <property name="title">
     <string>Teams</string>
    </property>
    <addaction name="actionTeams"/>
    <addaction name="actionAdd_Team"/>
   </widget>
   <widget class="QMenu" name="menuMatches">
    <property name="title">
     <string>Matches</string>
    </property>
    <addaction name="actionMatches"/>
    <addaction name="actionAdd_Match"/>
    <addaction name="actionRank"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
   <addaction name="menuTeams"/>
   <addaction name="menuMatches"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionSystem_Constant">
   <property name="text">
    <string>System Constant</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionTeams">
   <property name="text">
    <string>Teams</string>
   </property>
  </action>
  <action name="actionAdd_Team">
   <property name="text">
    <string>Add Team</string>
   </property>
  </action>
  <action name="actionMatches">
   <property name="text">
    <string>Matches</string>
   </property>
  </action>
  <action name="actionAdd_Match">
   <property name="text">
    <string>Add Match</string>
   </property>
  </action>
  <action name="actionRank">
   <property name="text">
    <string>Rank</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
